var/spack/repos/builtin/packages/parsec/package.py

```diff
@@ -104,15 +104,25 @@ def check(self):
                 warn += "https://bitbucket.org/icldistcomp/parsec/issues"
                 tty.msg(warn)
 
-    def test(self):
+    def test_parsec(self):
         """Compile and run a user program with the installed library"""
         with working_dir(join_path(self.install_test_root, "contrib/build_with_parsec")):
```

Comment:
Part of the conversion process also involves ensuring stand-alone tests do not attempt to build or run under the installation prefix.  So the test should start with copying the relevant files to the test stage directory before building the tests.